src/ServiceControl.Monitoring.Tests/QueueLength/QueueLengthDataConsumerTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿namespace ServiceControl.Monitoring.Tests.QueueLength
+{
+    using System;
+    using System.Collections.Generic;
+    using Monitoring.QueueLength;
+    using Newtonsoft.Json.Linq;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class QueueLengthDataConsumerTests
+    {
+        [Test]
+        public void It_extracts_sent_sequence()
```

Comment:
Why to we put JSON in here instead of serializing data using `Metrics` API? To ensure compatibility when the package is updated? IMO this test is fragile.